lib/core/reporters/helpers/process-aggregate.js

```diff
@@ -10,19 +10,19 @@ function normalizeRelatedNodes(node, options) {
       .forEach(checkRes => {
         checkRes.relatedNodes = checkRes.relatedNodes.map(relatedNode => {
           var res = {
-            html: relatedNode.source
+            html: relatedNode?.source ?? 'Undefined'
```

Comment:
I would have made this all lowercase so it looks like A Javascript stringification of a variable with a value of `undefined`, but nbd